lib/fizzy/types.hpp

```diff
@@ -190,7 +190,7 @@ struct Import
     } desc;
 };
 
-enum class ExportType : uint8_t
+enum class ExportKind : uint8_t
```

Comment:
This is actually called export type in the spec, isn't it? But I think kind is a better term.

Reply:
There's no name for it in Binary Format, but in Structure there's a notion of "External type" (the actual type of function/global etc.) https://webassembly.github.io/spec/core/syntax/types.html#external-types
which suggests maybe we should merge `ImportKind` and `ExportKind` into `ExternalKind`

Reply:
Yes, sounds good.